### ISSUE TITLE
OCR Language list is empty

### ISSUE THREAD

**Issue by denis-robel** [R0Wi/workflow_ocr#184]
the language list is empty for all users.
All I get is 'List is empty.'

Maybe that's why all users are are authenticated via ldap.

For local admin user the list working correctly.

**Comment by R0Wi** [comment]
Thanks for reporting this. Could you kindly share your `nextcloud.log` server logs and maybe some of your browsers DevTools logs, too?

**Comment by R0Wi** [comment]
@denis-robel did you already have the possibility to check some of your logs?

**Comment by R0Wi** [comment]
I think this error is indeed related to the fact if the user is admin or not. I was able to reproduce this issue by using a non-admin user and trying to edit/create a personal flow.

@denis-robel could you please confirm if my fix works for you, too by applying the following patch:

```bash
# Please adjust path according to your system
cd /var/www/html/nextcloud/apps/workflow_ocr
curl -s https://github.com/R0Wi/workflow_ocr/commit/9449c4fa961b7640c706cf634580256b525fec18.patch | patch -p1
```

**Comment by R0Wi** [comment]
Fixed in `v1.25.4` and `v1.24.7`
